src/SourceBuild/content/Directory.Build.props

```diff
@@ -62,6 +62,9 @@
        Keep in sync with props/targets in the Arcade.Sdk. -->
   <PropertyGroup Condition="'$(_SuppressSdkImports)' == 'true' or '$(SkipArcadeSdkImport)' == 'true'">
     <!-- RepoLayout.props -->
+    <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' != ''">$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)'))</NuGetPackageRoot>
+    <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == '' and '$(NUGET_PACKAGES)' != ''">$([MSBuild]::NormalizeDirectory('$(NUGET_PACKAGES)'))</NuGetPackageRoot>
```

Comment:
`NuGetPackageRoot` will be unset if `NuGetPackageRoot` and `NUGET_PACKAGES` are both unset.

Reply:
NuGetPackageRoot is defined by NuGet itself in NuGet.g.props.